lib/kitchen/driver/solariszone.rb

```diff
@@ -251,7 +267,7 @@ def configure_ssh
               sleep 10
             end
           end
-          raise Exception, return_value[:stdout] if return_value[:exit_code] != 0
+          raise Exception, return_value[:stdout] if return_value[:exit_code] != 1
```

Comment:
We are trying to make sure root is not a role anymore, right? In that case, grep should exit with 1